.php-cs-fixer.dist.php

```diff
@@ -45,6 +45,12 @@
     'nullable_type_declaration_for_default_null_value' => ['use_nullable_type_declaration' => true],
     'ordered_class_elements' => true,
     'ordered_imports' => ['sort_algorithm' => 'alpha', 'imports_order' => ['class', 'function', 'const']],
+    'phpdoc_order' => ['order' => ['param', 'throws', 'return', 'phpstan-param', 'psalm-param', 'phpstan-return', 'psalm-return']],
+    'phpdoc_separation' => ['groups' => [
+        ['var', 'phpstan-var', 'psalm-var'],
```

Comment:
Do we prefer
```
/**
 * @var Foo
 * @phpstan-var Foo<T>
 */
```
or
```
/**
 * @var Foo
 * 
 * @phpstan-var Foo<T>
 */
```

Reply:
I prefer to separate all the phpstan- or psalm- and group them together. Wdyt?

Reply:
We generally separate phpstan tag, but seems like for @phpstan-var we didnt.
But I can update the config.

Reply:
It seems more consistent to me.

Reply:
I opened https://github.com/sonata-project/dev-kit/pull/2166 then